scripts/image_upload_glance.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python2
+
+"""
+Script to upload iso/ova/qcow images to Glance server.
+This script requires that python-glanceclient be installed.
+
+pip install python-glanceclient
+
+Usage:
+1.scripts/image_upload_glance.py --image cfme-rhevm-5.8.2.1-1.x86_64.qcow2 \
+                                 --image_name_in_glance cfme-5821.qcow2 \
+                                 --provider glance11
+
+"""
+import argparse
+import sys
+
+from cfme.utils.conf import cfme_data, credentials
+from keystoneauth1 import loading
+from keystoneauth1 import session
+from glanceclient import Client
+
+
+def upload_to_glance(image, image_name_in_glance, provider):
+    """
+    Upload iso/qcow images to Glance.
+    """
+    api_version = '2'  # python-glanceclient API version
+    provider_dict = cfme_data['management_systems'][provider]
+
+    loader = loading.get_plugin_loader('password')
+    auth = loader.load_from_options(
+        auth_url=provider_dict['auth_url'],
+        username=credentials['cloudqe_rhos_default']['username'],
+        password=credentials['cloudqe_rhos_default']['password'],
+        tenant_name=credentials['cloudqe_rhos_default']['tenant'])
+    glance_session = session.Session(auth=auth)
+    glance = Client(api_version, session=glance_session)
+
+    # Two images on Glance could have the same name since Glance assigns them different IDS.
+    # So, we are running a check to make sure an image with the same name doesn't already exist.
+    for image in glance.images.list():
+        if image.name == image_name_in_glance:
+            print("Image already exists on Glance server")
+            sys.exit(127)
+
+    glance_img = glance.images.create(name=image_name_in_glance)
+    # Update image properties before uploading the image.
+    glance.images.update(glance_img.id, container_format="bare")
+    glance.images.update(glance_img.id, disk_format="qcow2")
```

Comment:
**Optional**

I think the disk format should be an option as well, so this supports more than just qcow2 image upload. We can default to qcow2 if not passed, would be ideal to detect the type based on the given image.